src/Spreadaccacadmey/SpreadAcademy.styled.js

```diff
@@ -0,0 +1,227 @@
+import styled from "styled-components";
+
+export const SpreadAcademyContainer = styled.section`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff6f6;
+
+  @media (max-width: 991px) {
+    justify-content: flex-start;
+  }
+`;
+export const SpreadAcademySection = styled.section`
+  background-color: #fff6f6;
+`;
+
+export const SpreadAcademyWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0px 26px 0px 0px;
+
+  @media (max-width: 991px) {
+    flex-direction: column;
+    padding: 0px 0px 0px 0px;
+  }
+`;
+
+export const Pstyle = styled.p`
+  font-size: 13px;
+  line-height: 1px;
```

Comment:
not 1px here give proper line height @PRABgit01

Reply:
done